assets/js/components/higherorder/withdata.js

```diff
@@ -151,7 +151,7 @@ const withData = (
 				'googlesitekit.moduleDataReset',
 				'googlesitekit.moduleDataResetHandler',
 				() => {
-					this.setState( { data: false } );
+					this.setState( { data: false, zeroData: false } );
```

Comment:
@tofumatt now that I see this in context, we're now resetting 2/3 of the keys in the `NewComponent` state - the other being `error`. It seems that `error` should probably also be reset - essentially resetting all of the component state when the date range changes. What do you think? cc: @felixarntz @adamsilverstein

Reply:
I was thinking that too, but I was just a bit wary to change more than needed. That said, resetting it shouldn't cause any issues. I'll try it out locally, but I think that's a good call and would prevent future issues similar to this one. 👍

Reply:
Just tested this locally and it worked fine without causing issues, so I've added `error: false` to the `setState` call here. Ready for another look!